scipy/signal/_ltisys.py

```diff
@@ -1829,6 +1839,7 @@ def lsim2(system, U=None, T=None, X0=None, **kwargs):
     >>> import numpy as np
     >>> from scipy.signal import bessel, lsim2
     >>> import matplotlib.pyplot as plt
+    >>> import warnings
```

Comment:
I might be missing something, but it seems you're not using this in the docstring - does it fail if it's not there? 🤔